index.html

```diff
@@ -11,21 +11,23 @@
       type="module"
     ></script>
   </head>
-  <body class="flex justify-center">
+  <body class="flex justify-center bg-gray-50">
     <div class="flex-grow max-w-2xl mt-10 px-4">
-      <header>
-        <h1 class="font-bold mb-10 text(center gray-600 xl)">
-          Koodikrapula.fi
-        </h1>
+      <header class="flex justify-center">
+        <a href="#">
+          <h1 class="font-bold mb-6 text(center gray-600 2xl) hover:bg-gray-100 hover:text-red-400 w-min p-3 rounded">
```

Comment:
Because the heading has bottom margin (`mb-6`), it makes the link's interactive area extend too low:

![image](https://user-images.githubusercontent.com/2226144/112727148-aafbec80-8f29-11eb-99c3-43d13ddc6f6b.png)

The yellow area (the heading's margin) in the image is clickable.

So, the margin class should be moved to the link.